### DISCUSSION TITLE
ZigbeeMQTT - Socat not enabled dopo cambio canale

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2253] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti, 
dopo aver variato il canale rete zigbee da 11 a 25 e riavviato zigbee e l'intero sistema, non mi sale più l'addon Zigbee2Mqtt.
```
502: Bad Gateway
```
e nei log 
```
[12:25:06] INFO: Preparing to start...
[12:25:07] INFO: Socat not enabled
```

**Comment by snakuzzo** [comment]
Prova a spegnere Z2M, togli alimentazione alla chiavetta, rialimenti e riavvii Z2M.
Se non risolvi, prova a dare un'occhiata [QUI](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/268)

Ricorda che cambiando canale dovrai rifare il pairing di tutti i device

**Reply by Matty1184** [reply]
riavviato. nulla
staccata usb, riattaccata, riavviato. nulla
premuto reset sulla scheda, riavviato nulla.

**Reply by snakuzzo** [reply]
Segui la issue che ti ho linkato

**Reply by Matty1184** [reply]
eliminato mosquitto, riavvato, intallato mosquitto nulla.
ho anche provato a elimare zigbee riavviare renstallare, nulla

**Reply by snakuzzo** [reply]
mi spiace, ma non so che dirti

**Reply by snakuzzo** [reply]
Lasciando stare HA OS, hai provato a tirare su al volo un'istanza di zigbee2mqtt (magari con Docker) ?

**Reply by Matty1184** [reply]
no, non ho ancora provato...
per info questo è il log ora
```
[17:57:14] INFO: Preparing to start...
[17:57:15] INFO: Socat not enabled
[17:57:17] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-01-08 17:57:23: Logging to console and directory: '/config/zigbee2mqtt/log/2023-01-08.17-57-23' filename: log.txt
Zigbee2MQTT:info  2023-01-08 17:57:23: Starting Zigbee2MQTT version 1.29.1 (commit #unknown)
Zigbee2MQTT:info  2023-01-08 17:57:23: Starting zigbee-herdsman (0.14.83)
Zigbee2MQTT:error 2023-01-08 17:57:24: Configuration is not consistent with adapter state/backup!
Zigbee2MQTT:error 2023-01-08 17:57:24: - PAN ID: configured=6754, adapter=6754
Zigbee2MQTT:error 2023-01-08 17:57:24: - Extended PAN ID: configured=00124b0024c2393f, adapter=00124b0024c2393f
Zigbee2MQTT:error 2023-01-08 17:57:24: - Network Key: configured=01030507090b0d0f00020406080a0c0d, adapter=01030507090b0d0f00020406080a0c0d
Zigbee2MQTT:error 2023-01-08 17:57:24: - Channel List: configured=25, adapter=11
Zigbee2MQTT:error 2023-01-08 17:57:24: Please update configuration to prevent further issues.
Zigbee2MQTT:error 2023-01-08 17:57:24: If you wish to re-commission your network, please remove coordinator backup at /config/zigbee2mqtt/coordinator_backup.json.
Zigbee2MQTT:error 2023-01-08 17:57:24: Re-commissioning your network will require re-pairing of all devices!
Zigbee2MQTT:error 2023-01-08 17:57:24: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-01-08 17:57:24: Failed to start zigbee
Zigbee2MQTT:error 2023-01-08 17:57:24: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-01-08 17:57:24: Exiting...
Zigbee2MQTT:error 2023-01-08 17:57:24: Error: startup failed - configuration-adapter mismatch - see logs above for more information
    at ZnpAdapterManager.determineStrategy (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:229:35)
    at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:63:26)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)
```

**Reply by snakuzzo** [reply]
È sbagliata la config. Non è consistente perché trova il backup. Rimuovi il backup dalla directory e riprova

**Reply by Matty1184** [reply]
eliminato il file config, riavviato e ora ok..
entro nell'interfaccia, cambio il canele da 1  a 25 prima di fare il paring con i dispositivi... stesso problema di prima!
502: bad gateway.
entro nel file config, riporto il canale a 11 e entro nell'interfaccia senza problemi.
dove sto sbagliando?

**Reply by snakuzzo** [reply]
Vedi il log ...se è quello di prima vuol dire che devi adeguare il canale direttamente in config

**Reply by Matty1184** [reply]
arrestato il servizio (con impostazione ch 11)
entrato nel file config, variato da 11 a 25, avviato il servizio
ma aimè errore 502.

per risolvere ho dovuto eliminare tutti i file nella cartella ad eccezione del configuration (che ho ovviamente modificato con il canel 25 prima di avviarlo.
ora si avvia regolarmente.
